tools/clang/test/HLSLFileCheck/hlsl/semantics/sv_barycentrics/barycentrics1.hlsl

```diff
@@ -1,4 +1,4 @@
-// RUN: %dxilver 1.1 | %dxc -E main -T ps_6_1 %s | FileCheck %s
+// RUN: %dxilver 1.6 | %dxc -E main -T ps_6_1 %s | FileCheck %s
```

Comment:
I was thinking that it might be better to add a variant that depends on 1.6 and checks for the new required functionality note for these barycentrics without losing the coverage on earlier validators. I'm not sure exactly how much we care about earlier validators, but we clearly care enough to skip these tests because they fail in the first place.

Reply:
So you are thinking of an extra run line that covers the older validator and makes sure the feature flag isn't set, or just doesn't look for it?  We could I guess if these are the only unit tests that use barycentric features.

Reply:
I was thinking of adding a custom CHECK for the additional functionality notes I added that is only invoked on a new RUN line that only runs when validator 1.6 is available. I'm not sure the pre-1.6 testing is terribly important though

Reply:
Yeah I've updated the tests, see if that's what you were thinking.  I think it was good suggestion to keep old run lines as well.  This way it actually tests the back-compat flags change.